src/hooks/swap/useSendSwapTransaction.tsx

```diff
@@ -36,6 +36,7 @@ export default function useSendSwapTransaction(
         callback: async function onSwap(): Promise<ExecTxResponse> {
           const approveTx = await approve({
             chainId: trade.fromToken.chainId as ChainId,
+            toChainId: trade.toToken.wrapped.chainId,
```

Comment:
Why trade.toToken.wrapped.chainId rather than trade.toToken.chainId similar to above?

Reply:
fixed